packages/bridge-status-controller/package.json

```diff
@@ -73,7 +73,7 @@
   "peerDependencies": {
     "@metamask/accounts-controller": "^27.0.0",
     "@metamask/network-controller": "^23.0.0",
-    "@metamask/transaction-controller": "^51.0.0"
+    "@metamask/transaction-controller": "^52.0.0"
```

Comment:
Nit: can we add a changelog entry for this breaking peer dependency bump

Reply:
Done 👍